src/algebra/direct_limit.lean

```diff
@@ -156,7 +156,7 @@ span_induction ((quotient.mk_eq_zero _).1 H)
           to_module_totalize_of_le hik hi,
           to_module_totalize_of_le hjk hj]⟩)
   (λ a x ⟨i, hi, hxi⟩,
-    ⟨i, λ k hk, hi k (dfinsupp.support_smul hk),
+    ⟨i, λ k hk, hi k (@dfinsupp.support_smul ι G _ _ _ _ _ _ _ _ _ hk),
```

Comment:
This is super annoying. My best guess is that `ι G` were inferred from a local typeclass instance, which is no longer possible because the local instance is `add_comm_group` but `support_smul` wants `add_comm_monoid`.

Reply:
It looks more like a problem with unfolding/typeclass inference: trying to hint `((show dfinsupp.support (a • x) ⊆ dfinsupp.support x, from dfinsupp.support_smul) hk)` gives an "unable to infer instance `has_scalar` on `a • x`, unless we mark `local attribute [reducible] direct_sum`.

Reply:
Thanks - that sounds like something to fix in direct_sum, which will want this treatment too